code_search/src/requirements.txt

```diff
@@ -1,10 +1,13 @@
 astor~=0.7.0
-apache-beam[gcp]~=2.6.0
+apache-beam[gcp]~=2.8.0
 Flask~=1.0.0
 nltk~=3.3.0
-nmslib~=1.7.0
+# TODO(jlewi): nmslib builds are failing on Dataflow workers with Apache beam 2.8.0.
+# We shouldn't need nmslib in the Dataflow jobs.
```

Comment:
Right, we shouldn't need nmslib in computing function embeddings. Presumably you've confirmed it works without it (noting is trying to import nmslib). In which case this can be removed.

Reply:
Done.